src/enemyc.cpp

```diff
@@ -642,176 +642,43 @@ void KEnemy::LoadEnemies(const std::string& fullPath, Raster* enemy_gfx)
         }
         std::istringstream iss(line);
 
-        KFighter fighter_loaded_from_disk;
-
-        // Enemy name
-        iss >> fighter_loaded_from_disk.name;
-
-        // Enemy index (ignored)
-        int index = 0;
-        iss >> index;
-
-        // x-coord of image in datafile
-        int imagefile_x_coord = 0;
-        iss >> imagefile_x_coord;
-
-        // y-coord of image in datafile
-        int imagefile_y_coord = 0;
-        iss >> imagefile_y_coord;
+        KFighter fighter;
+        iss >> fighter;
```

Comment:
This moved all the `iss >> fighter.XYZ;` calls into the KFigher class, since it should be responsible for its own reading/writing rules.